lms/templates/navigation.html

```diff
@@ -166,3 +184,6 @@ <h2 class="course-header"><span class="provider">${course.display_org_with_defau
 % endif
 
 <%include file="help_modal.html"/>
+<%block name="js_extra">
+<%static:js group='language_edx'/>
```

Comment:
@andy-armstrong can you suggest me a location where I can include this JS file ?